examples/curb-detection/lifelong_learning_bench/testalgorithms/rfnet/rfnet_algorithm.yaml

```diff
@@ -0,0 +1,69 @@
+    # currently the options of value are as follows:
+    #   1> "default": the dataset is evenly divided based train_ratio;
+    splitting_method: "default"
+
+  # algorithm module configuration in the paradigm; list type;
+  modules:
+    # type of algorithm module; string type;
+    # currently the options of value are as follows:
+    #   1> "basemodel": contains important interfaces such as train、 eval、 predict and more; required module;
+    - type: "basemodel"
+      # name of python module; string type;
+      # example: basemodel.py has BaseModel module that the alias is "FPN" for this benchmarking;
+      name: "BaseModel"
+      # the url address of python module; string type;
+      url: "./examples/curb-detection/lifelong_learning_bench/testalgorithms/rfnet/basemodel.py"
+      # hyperparameters configuration for the python module; list type;
+      hyperparameters:
+        # name of the hyperparameter; string type;
+        - learning_rate:
+            values:
+              - 0.0001
+    #  2> "task_definition": define lifelong task ; optional module;
+    - type: "task_definition"
+      # name of python module; string type;
+      name: "TaskDefinitionByOrigin"
+      # the url address of python module; string type;
+      url: "./examples/curb-detection/lifelong_learning_bench/testalgorithms/rfnet/task_definition_by_origin.py"
+      # hyperparameters configuration for the python module; list type;
+      hyperparameters:
+        # name of the hyperparameter; string type;
+        # origins of data; value is ["real", "sim"], this means that data from real camera and simulator.
+        - origins:
+            values:
+              - [ "real", "sim" ]
+    #  3> "task_allocation": allocate lifelong task ; optional module;
+    - type: "task_allocation"
+      # name of python module; string type;
+      name: "TaskAllocationByOrigin"
+      # the url address of python module; string type;
+      url: "./examples/curb-detection/lifelong_learning_bench/testalgorithms/rfnet/task_allocation_by_origin.py"
+      # hyperparameters configuration for the python module; list type;
+      hyperparameters:
+        # name of the hyperparameter; string type;
+        # origins of data; value is ["real", "sim"], this means that data from real camera and simulator.
+        - origins:
+            values:
+              - [ "real", "sim" ]
+    - type: "unseen_sample_recognition"
+      # name of python module; string type;
+      name: "UnseenSampleRecognitionByScene"
+      # the url address of python module; string type;
+      url: "./examples/curb-detection/lifelong_learning_bench/testalgorithms/rfnet/unseen_sample_recognition_by_scene.py"
+       # hyperparameters configuration for the python module; list type;
+      hyperparameters:
+        - model_path:
+            values:
+              - "./examples/curb-detection/lifelong_learning_bench/testalgorithms/rfnet/models/Epochofprose17.pth"
```

Comment:
The model file "Epochofprose17.pth" has no origin which should be provided in readme.